src/lib.rs

```diff
@@ -66,8 +66,9 @@
 //! # use tikv_client::{RawClient, Result};
 //! # use futures::prelude::*;
 //! # fn main() -> Result<()> {
-//! # futures::executor::block_on(async {
-//! let client = RawClient::new(vec!["127.0.0.1:2379"], None).await?;
+//! # use tikv_client::request::request_codec::RawApiV1;
+//! futures::executor::block_on(async {
+//! let client = RawClient::new(vec!["127.0.0.1:2379"], RawApiV1, None).await?;
```

Comment:
This might break the original API, we should add a default implementation for API V1.